File: mobile/modules/bluetooth-sdk/ios/Source/MentraBluetoothSDKDebug.swift
import Foundation

@MainActor
public enum MentraBluetoothSDKDebug {
    public static func setOtaVersionUrl(_ otaVersionUrl: String, on sdk: MentraBluetoothSDK) throws {
        try sdk.setOtaVersionUrl(otaVersionUrl)
    }

    public static func getOtaVersionUrl(on sdk: MentraBluetoothSDK) throws -> String {
        try sdk.getOtaVersionUrl()
    }
}

File: mobile/modules/bluetooth-sdk/android/src/main/java/com/mentra/bluetoothsdk/MentraBluetoothSdkDebug.kt
package com.mentra.bluetoothsdk

object MentraBluetoothSdkDebug {
    @JvmStatic
    fun setOtaVersionUrl(
        sdk: MentraBluetoothSdk,
        otaVersionUrl: String,
    ) {
        sdk.setOtaVersionUrl(otaVersionUrl)
    }

    @JvmStatic
    fun getOtaVersionUrl(sdk: MentraBluetoothSdk): String = sdk.getOtaVersionUrl()
}
